README.md

```diff
@@ -25,6 +25,11 @@ $ podman machine set --rootful
 $ podman machine start
 ```
 
+## ✍ Prerequisites
+
+The package `osbuild-selinux` or equivalent osbuild SELinux policies must be installed in the system running
+`bootc-image-builder`, even when running it as a container.
```

Comment:
bootc-image-builder doesn't run outside containers so the last part of the sentence might be confusing.

Reply:
I wil remove it then